src/main/java/io/lettuce/core/RedisClient.java

```diff
@@ -744,28 +744,28 @@ private Mono<SocketAddress> lookupRedis(RedisURI sentinelUri) {
         return Mono.usingWhen(
                 Mono.fromCompletionStage(() -> connectSentinelAsync(newStringStringCodec(), sentinelUri, timeout)), c -> {
 
-            String sentinelMasterId = sentinelUri.getSentinelMasterId();
-            return c.reactive().getMasterAddrByName(sentinelMasterId).map(it -> {
+                    String sentinelMasterId = sentinelUri.getSentinelMasterId();
```

Comment:
These are interesting. I assume that with the evolution of the Eclipse formatted plugin, some things changed. As we didn't touch that file too often, now these reformattings manifest.

Reply:
I think there are also cases where people thought they applied the formatter but they didn't for some reason.
Anyway I think this would help us keep the codebase clean.